internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -101,33 +102,45 @@ func NewApplicationSnapshotImage(ctx context.Context, fs afero.Fs, url string, p
 		os.Setenv("SIGSTORE_TRUST_REKOR_API_PUBLIC_KEY", "1")
 	}
 
-	policySources, err := fetchPolicySources(p)
-	if err != nil {
-		log.Debug("Failed to fetch the policy sources from the ECP!")
-		return nil, err
-	}
+	// Return an evaluator for each of these
+	for _, sourceGroup := range p.EnterpriseContractPolicySpec.Sources {
+		policySources, err := fetchPolicySources(&sourceGroup)
+		if err != nil {
+			log.Debugf("Failed to fetch the policy sources from the ECP source group '%s'!", sourceGroup.Name)
+			return nil, err
+		}
 
-	log.Debug("Policy source definitions fetched")
-	for _, policySource := range policySources {
-		policySourceJson, _ := json.Marshal(policySource)
-		log.Debugf("policySourceJson: %s", policySourceJson)
-	}
+		log.Debugf("Policy source definitions fetched for source group %s", sourceGroup.Name)
+		for _, policySource := range policySources {
+			policySourceJson, _ := json.Marshal(policySource)
+			log.Debugf("policySourceJson: %s", policySourceJson)
+		}
 
-	c, err := newConftestEvaluator(ctx, fs, policySources, ConftestNamespace, p)
-	if err != nil {
-		log.Debug("Failed to initialize the conftest evaluator!")
-		return nil, err
+		c, err := newConftestEvaluator(ctx, fs, policySources, ConftestNamespace, p)
+		if err != nil {
+			log.Debug("Failed to initialize the conftest evaluator!")
+			return nil, err
+		}
+
+		log.Debug("Conftest evaluator initialized")
+		a.Evaluators = append(a.Evaluators, c)
 	}
-	log.Debug("Conftest evaluator initialized")
-	a.Evaluator = c
 	return a, nil
 }
 
 // fetchPolicySources returns an array of policy sources
-func fetchPolicySources(spec *policy.Policy) ([]source.PolicySource, error) {
-	policySources := make([]source.PolicySource, 0, len(spec.Sources))
-	for _, sourceUrl := range spec.Sources {
-		url := source.PolicyUrl{Url: sourceUrl, Kind: source.PolicyKind}
+func fetchPolicySources(s *ecc.Source) ([]source.PolicySource, error) {
+	policySources := make([]source.PolicySource, 0, len(s.Policy)+len(s.Data))
```

Comment:
Log the name of the PolicySource being fetched?

Reply:
Actually I decided to take out the logging here since it's mostly duplicating the logging done in the calling function.